MIP0/mip0.md

```diff
@@ -464,19 +464,51 @@ The removal process begins once the community has agreed on the reasoning for re
 
 **Responsibilities**
 
--   Governance Facilitators are persons or teams responsible for operating a governance and voting frontend that is available to the entire community as well as venues the community uses for communications.
--   Governance Facilitators have the authority and responsibility to put proposals up for MKR vote for full community ratification, if they determine, with input from the community, that the proposals have fulfilled all requirements to be eligible for MKR voting.
--   Reference: [“Mandate: Interim Governance Facilitators”](https://forum.makerdao.com/t/mandate-interim-governance-facilitators/264).
-    
+The Governance Facilitator's responsibilities are defined as the following:
+
+Core Responsibilities
+- Responsible for ensuring the health and integrity of communication channels that are used for communication within MakerDAO. These tasks include moderation duties, establishing processes and social norms, and defending the channels from trolling and Sybil attacks.
+- Required to remain neutral and objective on issues outside the governance domain and focus on policy, procedure and facilitation.
```

Comment:
This blanket requirement for neutrality seems too strong to me. How about something like,
> When acting in the role of facilitator, required to remain neutral and objective on issues. When stating opinions, required to act outside the role of facilitator. When opinions could be construed as in conflict with governance responsibilities then shall recluse themselves.

OK, that's not very eloquent. Maybe you can say it better.

Reply:
Rich and I discussed this, and I spent some time thinking about it. I actually think it's important that the Facilitator is directed to act with neutrality in issues outside of the governance domain.

I think most of the reason it doesn't seem like a big deal now is because we haven't had any truly controversial and potentially community-splitting issues crop up. In the event of something like that happening, I think that having a facilitator with a history of neutrality is a requirement for navigating the issue successfully. One of the important roles is that of moderation of discussion in venues (both calls and forum.) If there is any suspicion that the facilitator is moderating to favour one side of the discussion then all hell (rightly) breaks loose.

Rich made the point that as we onboard more facilitators, the neutrality requirements can be be relaxed somewhat (as facilitators can take either side of an issue.) Imo that could be a modification made at a later date.

On a personal note: While I'd like to be able to communicate my opinions on monetary policy and the future of the protocol there are others with deep insight into these issues and I certainly have enough opinions relating to governance to share instead.

Reply:
Hm, okay